src/Raven.Client/Http/OAuth/ApiKeyAuthenticator.cs

```diff
@@ -176,7 +176,13 @@ private string[] ExtractApiKeyAndSecret(string apiKey)
             var apiKeyParts = apiKey.Split(new[] { '/' }, StringSplitOptions.None);
             if (apiKeyParts.Length > 2)
             {
-                apiKeyParts[1] = string.Join("/", apiKeyParts.Skip(1));
+                var secretIndex = 1;
+                if (apiKeyParts[0] == "Raven" && apiKeyParts.Length > 3)
```

Comment:
I think that 'Raven/' is already reserved: https://github.com/ravendb/ravendb/blob/v4.0/src/Raven.Server/Web/Authentication/AdminApiKeysHandler.cs#L26